README.md

```diff
@@ -14,7 +14,18 @@ This is the repository for recommended [Helm](https://helm.sh/) charts for runni
 > [!NOTE] 
 > Arrow usage is in beta, please use at your own risk. Reach out if you have any issues.
 
-In order to use an arrow trace collector, you will need to build your own custom image. We have supplied a collector builder config below. Once an image is a available, simply apply your desired helm chart with the values.yaml AND the arrow.yaml in the respective chart. Make sure to replace the image in arrow.yaml with your custom built image.
+In order to use an arrow trace collector, you can use  (1) the prebuilt image available via the Github Container Registry (GHCR) or you may (2) build your own custom image.
+
+### 1. Use the prebuilt Docker image
+1. We have built a Docker image using the example [build config](https://github.com/lightstep/otel-collector-charts/blob/main/arrow/otelcolarrow-build.yaml)
```

Comment:
why don't we just modify this in the chart's arrow.yaml?

Reply:
Like..in here? 
https://github.com/lightstep/otel-collector-charts/blob/main/charts/kube-otel-stack/arrow.yaml 

or
https://github.com/lightstep/otel-collector-charts/blob/main/charts/otel-cloud-stack/arrow.yaml?

Reply:
Agreed, let's reference the published arrow image in both charts

Reply:
To be clear, you're referring to changing these lines, right?

https://github.com/lightstep/otel-collector-charts/blob/main/charts/kube-otel-stack/arrow.yaml#L7

https://github.com/lightstep/otel-collector-charts/blob/main/charts/otel-cloud-stack/arrow.yaml#L7

Reply:
yep! I think there are a few image references if i'm not mistaken

Reply:
Only found 2 references, but updated:
1. https://github.com/lightstep/otel-collector-charts/pull/65/files#diff-93cd8830725172b53879f9086350332ea8f25b6be5645cb02ff8b203e6a4263fR7
2. https://github.com/lightstep/otel-collector-charts/pull/65/files#diff-abe71beffe980586767aa6fea908bc97020bf1ca0041ee92a7df7f024f634945R7